### ISSUE TITLE
Proposed description of "Mankind 2.0" campaign (draft README.md)

### ISSUE THREAD

**Issue by georgeshk** [humanetech-community/humanetech-community-awareness#7]
| Field | Description |
| :---: | :--- |
| `Name` | Mankind 2.0 |
| `Summary` | Series of short, dramatic/impactful videos that illustrate the pervasive use of technology and rapid, drastic changes in our society, leading to introspection and pondering the question: "Are we really fine with this?" |
| `Goals` | <ul><li>Shock the audience</li><li>Lead to introspection and questioning</li><li>Make the audience think and reach their own conclusions</li><li>Provoke, rather than convince</li></ul> |
| `Audience` | All ages from 16 |
| `Success criteria` | <ul><li>Number of views</li><li>[Measure of positive impact to be defined]</li></ul> |
| `Description` | The campaign will feature short videos (2-3 minutes each) that aim to be dramatic and impactful through the combined use of words, cinematography and music.<br><br>It does not aim to explicitly pursue a specific agenda, push on the audience a clear message, or impose a "lecture". Rather, it aims to provoke introspection by telling provocative stories, the conclusions of which are left to the audience.<br><br>Why is it important to adopt a "neutral" approach based on providing factual observations in a spectacular and dramatic form? Because this campaign is based on the principle that we will not change our behaviors on the mere basis of facts and reason if we have no motivation to challenge our thinking in the first place and take a step back from our habits. The dramatic videos aim to shock and leave a strong impression that ideally will lead to wondering where each of us stands in all this. |
| `Deliverables` | [mankind-2.0-deliverables](deliverables.md) |
| `Strategy` | <ul><li>To achieve maximum impact, we will need to write high-quality, original scripts, and rely on professional video producers to make sure the final product is impeccable in every little detail (one small mistake and "haters" will jump on it).</li><li>For maximum distribution, refer to the general [marketing](campaigns/strategy/marketing) section.</li><ul> |
| `Funding` | (optional) Financial requirements, required budget, ways to obtain funds |
| `Milestones` | (optional) Summary of past and future milestones for the campaign |
| `Resources` | (optional) Links to relevant subfolders, files and external information |

**Comment by patmatsu** [comment]
Sounds great; I can help with writing, editing, and proofreading.

**Comment by georgeshk** [comment]
Hi @patmatsu

Great! I am not a native English speaker, so that would help a lot. Also, please feel free to express your views on the content itself.





Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by patmatsu** [comment]
Thank you, @georgeshk. I will do my best to contribute.

Have you by chance seen this [video](https://www.youtube.com/watch?v=tf9ZhU7zF8s) by Max Stossel, a member of the CHT team? I believe he helped establish the forum.

**Comment by patmatsu** [comment]
I bring up the video because it is very good in some ways and not good in other ways. I therefore think it might be helpful in establishing what qualities or characteristics are desirable and what aren't.

**Comment by patmatsu** [comment]
I agree with and support everything you say. Thank you, @georgeshk :-)

**Comment by georgeshk** [comment]
(Same as with Mankind 2.0 theme proposal). I know we are still very few to constitute a representative segment of the community, but it seems we all agree on this (unless you have any further comment), plus we started with concepts under this campaign.

Proposing to close this issue and move campaign description to the "Code" section. Agreed?

**Comment by aschrijver** [comment]
Let's leave open a bit longer, but create the document (as I PM'ed about just now)

**Comment by georgeshk** [comment]
Just created a pull request so that we can update the README file and close this issue.
